tools/base/fpgad/ap6.c

```diff
@@ -285,7 +288,8 @@ void *ap6_thread(void *thread_context)
 		ret = read_bitstream(c->config->null_gbs[i], &null_gbs_info);
 		if (ret < 0) {
 			dlog("ap6[%i]: \tfailed to read bitstream\n", c->socket);
-			free((void *)null_gbs_info.data);
+			if (null_gbs_info.data)
+				free((void *)null_gbs_info.data);
```

Comment:
do we need to set null_gbs_info.data to NULL here?

Reply:
safe to set to NULL